utils/chaos-recommender/Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+    docker build -t engineered_chaos .
+
+run:
```

Comment:
@mudverma we might want to add instructions around building and running the containerized version in the readme. Thoughts?

Reply:
@chaitanyaenr  Updated the instructions to run with docker